### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by sajid-munawar**
**In this milestone**
- Add styles

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -2,16 +2,14 @@ import { Link } from 'react-router-dom';
...
-        <Link to="/categories">Categories</Link>
-      </nav>
+    <div className="nav-container">
+      <div className="navbar">
+        <h1 className="Bookstore-CMS">Bookstore CMS</h1>
+        <nav>
+          <Link to="/" className="BOOKS">Books</Link>
+          <Link to="/categories" className="CATEGORIES">Categories</Link>
+        </nav>
+      </div>
```
Comment (ahmetbozaci) [inline src/components/Navbar.js]: - [ ] The design should be the same. Kindly update it. [Requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md#:~:text=Your%20final%20design%20should%20match%20Zeplin%27s%20design.)

image

___
### ORIGINAL DESIGN
image

File: src/components/App.css [hunk 2]
```diff
@@ -1,3 +1,152 @@
...
+  align-items: center;
+  gap: 5%;
+}
+
+a {
+  text-decoration: none;
+}
+
+.Bookstore-CMS {
+  font-family: 'Montserrat', sans-serif;
```
Comment (ahmetbozaci) [inline src/components/App.css:36]: - [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: src/components/App.css [hunk 3]
```diff
@@ -1,3 +1,152 @@
...
+.nav-container {
+  width: 87.5rem;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+}
+
+.navbar {
+  display: flex;
```
Comment (ahmetbozaci) [inline src/components/App.css:17]: - [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: src/components/App.css [hunk 4]
```diff
@@ -1,3 +1,152 @@
...
+.Title {
+  width: 12.313rem;
+  height: 1.813rem;
+  margin: 0.188rem 20.188rem 0 0;
+  font-family: 'Roboto Slab', serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```
Comment (ahmetbozaci) [inline src/components/App.css:85]: - [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @sajid-munawar  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- You have one commit for everything. For best practices, you should commit often. [Git Commit Best Practices](https://www.perforce.com/blog/vcs/git-best-practices-git-commit)
___

### Cheers and Happy coding! 💻 👏
almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: src/components/Book.js [hunk 5]
```diff
@@ -3,19 +3,51 @@ import { useDispatch } from 'react-redux';
...
+      <div className="f-column">
+        <span className="completed">Current Chapter</span>
+        <span>
+          Chapter
+          {' '}
+          { chapter }
+        </span>
+        <button type="button" className="ADD-BOOK update-progress">Update Progress</button>
+      </div>
+
```
Comment (descholar-ceo) [inline src/components/Book.js:50]: - [ ] Would you refactor your design to match the project requirement on this bottom form? The form of adding new book is still need to be worked on,

### Original design
Screenshot from 2022-07-29 07-01-41

### Your design
Screenshot from 2022-07-29 06-57-20

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @sajid-munawar , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
👍🏽 Impemented most of the requested changes ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js [hunk 6]
```diff
@@ -3,25 +3,51 @@ import { useDispatch } from 'react-redux';
...
-        {title}
-      </h2>
-      <h4>
-        Author :
-        {' '}
-        {author}
-        {' '}
-      </h4>
-      <button type="button" onClick={() => handleClick()}>Remove</button>
+    <div className="BOOK_CONTAINER">
```
Comment (Ismailco) [inline src/components/Book.js:13]: - [ ] The book list should be centered on the page, also kindly add the profile icon in the header so your design looks like the Zeplin one

|Your design|Zeplin design|
|-|-|
|Screenshot_20220729_064842|Screenshot_20220728_202309|

**Review by Ismailco — APPROVED** [review]
### Hi @sajid-munawar, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**